sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlockBlobAsyncClient.java

```diff
@@ -312,9 +311,8 @@ private String getBlockID() {
      *
      * @return A reactive response signalling completion.
      */
-    public Mono<Void> stageBlockFromURL(String base64BlockID, URL sourceURL, BlobRange sourceRange) {
-        return stageBlockFromURLWithResponse(base64BlockID, sourceURL, sourceRange, null,
-            null, null).flatMap(FluxUtil::toMono);
+    public Mono<VoidResponse> stageBlockFromURL(String base64BlockID, URL sourceURL, BlobRange sourceRange) {
```

Comment:
I think this change should be reverted. The non-max overload shouldn't return a Response, so it should stay Mono<Void>